R/setkey.R

```diff
@@ -9,7 +9,14 @@ setkey <- function(x, ..., verbose=getOption("datatable.verbose"), physical=TRUE
 
 # FR #1442
 setindex <- function(...) setkey(..., physical=FALSE)
-setindexv <- function(...) setkeyv(..., physical=FALSE)
+setindexv <- function(x, cols, verbose=getOption("datatable.verbose")) {
+  if (is.list(cols)) {
+    sapply(cols, setkeyv, x = x, verbose = verbose, physical = FALSE)
+    return(invisible(x))
+  } else {
+    setkeyv(x, cols, verbose, physical=FALSE)
```

Comment:
I would use `verbose=verbose` as this argument can easily shift in future if we add another one in front. But as of now works well.

Reply:
Good catch, will fix.